places/Cargo.toml

```diff
@@ -3,6 +3,11 @@ name = "places"
 version = "0.1.0"
 authors = []
 
+[features]
+bundled = ["rusqlite/bundled"]
```

Comment:
Ah, I see -- this probably explains the Travis error.  Can you investigate?

Reply:
Right, we're doing an --all-features in the travis tests. Ugh.